wordpress-mobile/WordPress-Android
# Layouts picker scroll position

Fixes layout picker scroll position issue

Fixes #14364 

To test:

- scroll vertically to the bottom of this screen, 
- then scroll horizontally to the end of the Home pages list, then select the last layout, like the Raise Your Helping Hand
- and either scroll up to the top and back down to the bottom to see that the horizontal scroll position was reset to the left.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
